Makefile

```diff
@@ -33,6 +33,7 @@ ifeq ($(PLATFORM), bittboy)
 	STRIP		:= arm-linux-strip
 	SYSROOT		:= $(shell $(CC) --print-sysroot)
 	CFLAGS		:= $(shell $(SYSROOT)/usr/bin/sdl-config --cflags)
+	CFLAGS		+= -DSCREEN_SCALE=1
```

Comment:
Move this change to the previous commit, as it is unrelated to the tearing fix.

Reply:
Feel free to move it around.